rust/processor/src/schema.rs

```diff
@@ -518,6 +539,80 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    fungible_asset_activities (transaction_version, event_index) {
+        transaction_version -> Int8,
+        event_index -> Int8,
+        #[max_length = 66]
+        owner_address -> Varchar,
+        #[max_length = 66]
+        storage_id -> Varchar,
+        #[max_length = 1000]
+        asset_type -> Varchar,
+        is_frozen -> Nullable<Bool>,
+        amount -> Nullable<Numeric>,
+        #[sql_name = "type"]
+        type_ -> Varchar,
+        is_gas_fee -> Bool,
+        #[max_length = 66]
+        gas_fee_payer_address -> Nullable<Varchar>,
+        is_transaction_success -> Bool,
+        #[max_length = 1000]
+        entry_function_id_str -> Nullable<Varchar>,
+        block_height -> Int8,
+        #[max_length = 10]
+        token_standard -> Varchar,
+        transaction_timestamp -> Timestamp,
+        inserted_at -> Timestamp,
+    }
+}
+
+diesel::table! {
+    fungible_asset_balances (transaction_version, write_set_change_index) {
+        transaction_version -> Int8,
+        write_set_change_index -> Int8,
+        #[max_length = 66]
+        storage_id -> Varchar,
+        #[max_length = 66]
+        owner_address -> Varchar,
+        #[max_length = 1000]
+        asset_type -> Varchar,
+        is_primary -> Bool,
+        is_frozen -> Bool,
+        amount -> Numeric,
+        transaction_timestamp -> Timestamp,
+        #[max_length = 10]
+        token_standard -> Varchar,
+        inserted_at -> Timestamp,
+    }
+}
+
+diesel::table! {
+    fungible_asset_metadata (asset_type) {
+        #[max_length = 1000]
+        asset_type -> Varchar,
+        #[max_length = 66]
+        creator_address -> Varchar,
+        #[max_length = 32]
+        name -> Varchar,
+        #[max_length = 10]
+        symbol -> Varchar,
+        decimals -> Int4,
+        #[max_length = 512]
+        icon_uri -> Nullable<Varchar>,
```

Comment:
Technically not optional but can be empty strings. What's the policy here on using Nullable vs just leaving it empty?